src/support.cpp

```diff
@@ -114,7 +116,7 @@ void AreaSupport::splitGlobalSupportAreasIntoSupportInfillParts(SliceDataStorage
 
 void AreaSupport::generateSupportInfillFeatures(SliceDataStorage& storage)
 {
-    AreaSupport::prepareInsetsAndInfillAreasForForSupportInfillParts(storage);
+    AreaSupport::prepareSupport(storage);
```

Comment:
Might as well be renamed to `prepare` then.
But maybe it's better to specify what it's preparing. In this case it seems to already actually generate the walls, through `wall_maker.generateToolpaths`?